<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Paths</class>
 <widget class="QDialog" name="Paths">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>236</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Paths</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QPushButton" name="button_add">
     <property name="toolTip">
      <string>Add</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="list-add">
       <normaloff/>
      </iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLineEdit" name="edit">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>00000</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="button_remove">
     <property name="toolTip">
      <string>Remove</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="list-remove">
       <normaloff/>
      </iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="button_clear">
     <property name="toolTip">
      <string>Clear</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="edit-delete">
       <normaloff/>
      </iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="5">
    <widget class="QTableWidget" name="table"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
